cmd/check-operator/upgrade/upgrade.go

```diff
@@ -157,6 +158,17 @@ func (o *Options) getV2Status() (common.StatusWrapper, error) {
 	return common.NewV2StatusWrapper(datadogAgent), nil
 }
 
+func isReconcileError(conditions []metav1.Condition) bool {
+	for _, condition := range conditions {
+		if (condition.Type == "DatadogAgentReconcileError" && condition.Status == metav1.ConditionTrue) ||
+			(condition.Type == "AgentReconcile" && condition.Status == metav1.ConditionFalse) ||
+			(condition.Type == "ClusterAgentReconcile" && condition.Status == metav1.ConditionFalse) {
```

Comment:
we also have `ClusterChecksRunnerReconcile` that can be check when cluster check runner option is enable

Reply:
thanks for pointing it out, added!